plugins/postcss-cascade-layers/src/index.ts

```diff
@@ -44,9 +45,8 @@ const creator: PluginCreator<undefined> = () => {
 
 					// go through the unlayered rules, clone, and delete from top level atRule
 					atRule.each((node) => {
-						if (node.type == 'rule') {
-							implicitLayer.append(node.clone());
-							node.remove();
+						if (node.type == 'atrule' && node.name === 'layer') {
+							return;
```

Comment:
@oluoluoxenfree I think this bit was changed somewhere in the merges.
Unsure if it was intentional, but it broke the tests :)